docs/source/quickstart.rst

```diff
@@ -1,2 +1,154 @@
-Quickstart
-==========
+.. _quickstart:
+
+#############
+ Quick Start
+#############
+
+This page will guide you to train a GAN to remove glasses for human
+faces, and a DDPM to add glasses on human faces.
+
+.. _quickstart-install:
+
+**************
+ Installation
+**************
+
+Clone this repo:
+
+.. code:: bash
+
+   git clone --recursive https://github.com/jolibrain/joliGEN.git
+   cd joliGEN
+
+Install PyTorch and other dependencies (torchvision, visdom) with:
+
+.. code:: bash
+
+   pip install -r requirements.txt --upgrade
+
+*****************************************************
+ Train a GAN for unpaired dataset and object removal
+*****************************************************
+
+.. _quickstart-gan-dataset:
+
+Download the Dataset
+====================
+
+Download the dataset (3.3 Gb) and unzip it:
+
+.. code:: bash
+
+   wget https://www.deepdetect.com/joligen/datasets/noglasses2glasses_ffhq.zip
+   unzip noglasses2glasses_ffhq.zip
```

Comment:
Add a line to put the dataset into a dir, and then point to the dir with `--dataroot` it'll be clearer, add  an `mkdir checkpoints` as well.